tests/Language/AbstractTranslationTestCase.php

```diff
@@ -47,6 +47,7 @@ abstract class AbstractTranslationTestCase extends TestCase
         GreekTranslationTest::class              => 'el',
         SpanishTranslationTest::class            => 'es',
         FarsiTranslationTest::class              => 'fa',
+        GalicianTranslationTest::class           => 'gl',
         FrenchTranslationTest::class             => 'fr',
```

Comment:
Please fix this error:
```diff
1) Translations\Tests\AutoReview\TranslationsCodeTest::testLocalesArrayAreArrangedByLocaleCode
The `AbstractTranslationTestCase::$locales` property should be arranged by locale code.
Failed asserting that two arrays are identical.
--- Expected
+++ Actual
@@ @@
     'Translations\Tests\GreekTranslationTest' => 'el'
     'Translations\Tests\SpanishTranslationTest' => 'es'
     'Translations\Tests\FarsiTranslationTest' => 'fa'
+    'Translations\Tests\GalicianTranslationTest' => 'gl'
     'Translations\Tests\FrenchTranslationTest' => 'fr'
-    'Translations\Tests\GalicianTranslationTest' => 'gl'
     'Translations\Tests\HungarianTranslationTest' => 'hu'
     'Translations\Tests\IndonesianTranslationTest' => 'id'
     'Translations\Tests\ItalianTranslationTest' => 'it'

/home/runner/work/translations/translations/tests/AutoReview/TranslationsCodeTest.php:34
phpvfscomposer:///home/runner/work/translations/translations/vendor/phpunit/phpunit/phpunit:106

FAILURES!
Tests: 3, Assertions: 73, Failures: 1.
Error: Process completed with exit code 1.
```
https://github.com/codeigniter4/translations/actions/runs/7944239245/job/21691254003?pr=437

Reply:
I repair this error